TrainingHW7/TrainingHW7/Comparer/SortByNameLength.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingHW7.Comparer
+{
+    class SortByNameLength : IComparer<Employee>
+    {
+        public int Compare(Employee x, Employee y)
+        {
+            if ((x.FirstName.Length + x.LastName.Length) < (y.FirstName.Length + y.LastName.Length))
```

Comment:
better use Fullname property in Employee class
`public string Fullname {get;set;} = Firstname + Lastname;`
and do smth with it.